packages/core/src/components.d.ts

```diff
@@ -90,6 +91,7 @@ export { SkiplinkClickEvent } from "./components/skiplink/skiplink.interfaces";
 export { SlideToggleActiveEvent } from "./components/slide-toggle/slide-toggle.interfaces";
 export { SurveyRatingCloseEvent, SurveyRatingSubmitEvent } from "./components/survey-rating/survey-rating.interfaces";
 export { TabsSwitchEvent } from "./components/tabs/tabs.interfaces";
+export { TooltipPosition } from "dso-toolkit";
```

Comment:
Hier klopt iets nog niet zie ik nu. Er wordt een type uit `dso-toolkit` geexporteerd terwijl we dat (nog?) niet doen.

Ik denk dat `TooltipPosition` gekopieerd moet worden naar naar waar de functional component staat: `/packages/core/src/functional-components/tooltip/tooltip.interfaces.ts`